### PR TITLE
Add log-opts to Splunk daemon.json example

### PR DESCRIPTION
**Pull request by gbarr01**
Enhance Splunk example submitted by @anweiss.

### CODE REVIEW

**Comment by johndmulhausen** [comment]
@anweiss Can you review @gbarr01's changes here?

**Comment by anweiss** [comment]
The line further down that reads:

> You can use the `--log-opt NAME=VALUE` flag to specify these additional Splunk

... should make the distinction more clear between passing `--log-opt` flags to individual containers vs. configuring them in the daemon.

Instead, the line should read more like:

> Refer to the listing below for properties to include in the `log-opts` object in `daemon.json`. You can also pass one or more `--log-opt NAME=VALUE` flags to an individual container in order to specify the log driver configuration on a per-container basis.

File: engine/admin/logging/splunk.md [hunk 1]
```diff
@@ -11,33 +11,42 @@ The `splunk` logging driver sends container logs to
 in Splunk Enterprise and Splunk Cloud.
 
 ## Usage
+You can configure Docker logging to use the `splunk` driver by default or on a
```
Comment (mstanleyjones) [inline engine/admin/logging/splunk.md]: nit: newline

File: engine/admin/logging/splunk.md [hunk 2]
```diff
@@ -11,33 +11,42 @@ The `splunk` logging driver sends container logs to
...
+  docker run --log-driver=splunk --log-opt splunk-token=VALUE --log-opt splunk-url=VALUE ...
+  ```
 
 ## Splunk options
 
-You can use the `--log-opt NAME=VALUE` flag to specify these additional Splunk
-logging driver options:
+The following properties let you configure the splunk logging driver.
+
+- To configure the `splunk` driver across the Docker environment, edit daemon.json with `"log-opts": {"NAME": "VALUE", ...}`.
```
Comment (mstanleyjones) [inline engine/admin/logging/splunk.md]: nit: wrapping

**Pull request merged**